Configurateur/front/lib/screens/contact/contact.dart

```diff
@@ -96,8 +102,8 @@ class _ContactPageState extends State<ContactPage> {
             ),
           ),
           const SizedBox(height: 100),
-          Padding(
-            padding: const EdgeInsets.only(right: 400, left: 400),
+          FractionallySizedBox(
+            widthFactor: 0.8,
```

Comment:
why you don't add de widthFactor in the globalStyle or is own style file like the others ?